### ISSUE TITLE
Nipple discharge, abnormal

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10070]
**Preferred term label:**
Nipple discharge, abnormal

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0149741

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Intraductal papilloma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000062
- label: Abnormal nipple discharge
- parents: [Abnormal breast physiology (HP:0031094)](https://hpo.jax.org/app/browse/term/HP:0031094)
- pmid: PMID:36335926
- def: Abnormal fluid that seeps out of the nipple of the breast. Nipple discharge is categozied as lactational, physiologic, or pathologic.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10070
- synonyms:
